actionpack/lib/action_controller/renderer.rb

```diff
@@ -71,6 +71,18 @@ def initialize(controller, env, defaults)
     end
 
     # Render templates with any options from ActionController::Base#render_to_string.
+    #
+    # The primary options are:
+    # * <tt>:partial</tt> - See <tt>ActionView::PartialRenderer</tt> for details.
+    # * <tt>:file</tt> - Renders an explicit template file. Add <tt>:locals</tt> to pass in, if so desired.
+    # * <tt>:inline</tt> - Renders an inline template.
+    # * <tt>:plain</tt> - Renders provided text and sets the content type as <tt>text/plain</tt>.
+    # * <tt>:html</tt> - Renders the provided HTML safe string, otherwise
+    #   performs HTML escape on the string first. Sets the content type as <tt>text/html</tt>.
+    # * <tt>:body</tt> - Renders provided text and sets content type of <tt>text/plain</tt>.
```

Comment:
This doesn't line up with the guides section on rendering. See http://guides.rubyonrails.org/layouts_and_rendering.html#rendering-raw-body

Reply:
This is technically true. I’m fine with it.